getting-started/adding-telerik-ui-to-an-application/installation-adding-to-application-create-application-and-adding-control.md

```diff
@@ -1,90 +1,112 @@
 
-* Open [Microsoft Visual Studio 2012]({%slug installation-system-requirements%}).
-          	
-* Create a new Silverlight Application.
-![Common Installing Creating Application 011](images/Common_InstallingCreatingApplication_011.png)
+	#### __Figure 3: Adding references to the charting controls__
+	![Common Installing Creating Application 013 WPF](images/installation-adding-to-application-create-application-and-adding-control-3.png)	
 
->You can choose __.Net Framework 4.0__ or __.Net Framework 4.5__ for your application.
+	> This step is optional and you will need it only if use controls that are not defined in Telerik.Windows.Controls.dll.
 
-## Dragging Controls from Visual Studio Toolbox
+## Creating Application and Manually Referencing the Telerik Assemblies
 
-* Add the Telerik Silverlight controls to your [toolbox]({%slug installation-adding-to-vs-2012-toolbox%}). 
+1. Open [Microsoft Visual Studio]{% if site.site_name == 'Silverlight' %}({%slug installation-system-requirements%}){% else %}({%slug installation-system-requirements-wpf%}){% endif %}.
+          	
+2. Create a new {{ site.platform_name }} application.
+	
+	#### __Figure 4: Creating new {{ site.platform_name }} application__
+	![Creating new application](images/installation-adding-to-application-create-application-and-adding-control-4.png)
 
->If you wonder which control in which assembly belongs to, read the main topic about [Controls Dependencies]({%slug installation-installing-controls-dependencies%}).
+3. Add references to the Telerik assemblies. There are two options to do this.
+	
+	* [Drag a Telerik control from the toolbox](#dragging-controls-from-visual-studio-toolbox). When the control gets dropped onto the Visual Studio designer's surface, Visual Studio will automatically add the corresponding references.
+	
+	* Manually reference the Telerik dlls from the UI for {{ site.platform_name }} installation folder.	The binaries are located in the *UI for {{ site.platform_name }} installation folder/Binaries/{% if site.site_name == 'WPF' %}WPF40/{% endif %}*
+	
+### Dragging Controls from Visual Studio Toolbox
 
-* Open MainPage.xaml of the project you have previously created.
+* Add the Telerik {{ site.platform_name }} controls to the [Visual Studio Toolbox]({%slug installation-adding-to-vs-2015-toolbox-wpf%}).
 
-* Select any control in the Visual Studio toolbox.
+	>tip If you wonder which control in which assembly belongs to, read the main topic about {% if site.site_name == 'Silverlight' %}[Controls Dependencies]({%slug installation-installing-controls-dependencies%}){% else %}[Controls Dependencies]({%slug installation-installing-controls-dependencies-wpf%}){% endif %}.
```

Comment:
We should improve the wording here:
"If you're wondering which controls belongs to which assembly..."